src/main/resources/hudson/plugins/report/genericchart/GenericChartProjectAction/floatingBox.jelly

```diff
@@ -22,14 +22,18 @@
                         label: "${chart.title}",
                                 fillColor: "${chart.color}",
                                 strokeColor: "${chart.color}",
-                                pointColor: "${chart.color}",
-                                pointStrokeColor: "#fff",
+                                pointStrokeColor: "#808080",
                                 pointHighlightFill: "#fff",
-                                pointHighlightStroke: "rgba(180,180,180,1)",
+                                pointHighlightStroke: "rgba(0,0,0,1)",
```

Comment:
Why this change?

Reply:
StrokeColor is changed to #808080 so if it would stay without change the color difference between highlighted and normal strokeColor would  be small. Which is bad because of readability